### PR TITLE
Obtain eltype of an ImageHDU

### LINKED ISSUE #147: Obtain eltype without reading the array
Since images have the element type in the header (and this is something that CFITSIO converts to a Julia type), it should be possible to obtain the element type of an image HDU using something like this :

```julia
julia> FITS("V_kw.fits", "r") do f
           eltype(f[1])
       end
```

Currently this returns `Any`, but a suitable method may be added to return the correct element type.

### PR DESCRIPTION
**Pull request by jishnub**
Fixes #147 

Now 

```julia
julia> FITS("temp.fits", "w") do f
       write(f, zeros(Int32, 2))
       end

julia> FITS("temp.fits", "r") do f
       eltype(f[1])
       end
Int32
```

### CODE REVIEW

**Review by mileslucas — APPROVED** [review]
Thanks, this looks nice and concise! I'll approve for now but I'll wait for another pair of eyes before merging.

**Review by giordano — APPROVED** [review]

**Pull request merged**
